deploy/manifests/BUILD.bazel

```diff
@@ -83,3 +84,12 @@ filegroup(
     tags = ["automanaged"],
     visibility = ["//visibility:public"],
 )
+
+modify_file(
+    name = "kustomize-base-regular",
+    src = "//deploy:kustomize-base",
+    out = "kustomize.yaml",
+    prefix = "",
+    suffix = "\n  - cert-manager.yaml",
```

Comment:
Is it expected that a user fetches this file from GitHub and places both the kustomize.yaml file as well as the cert-manager.yaml file into the same directory in order to consume this?